src/qsql_ibpp.h

```diff
@@ -29,7 +29,7 @@
 #include <QtSql/qsqlresult.h>
 #include <QtSql/qsqldriver.h>
 #include <qsqldriverplugin.h>
-#include "qsqlcachedresult_p.h"
+#include "src/qt_5_9_4/qsqlcachedresult_p.h"
```

Comment:
Copying Qt source files seems strange to me.

Reply:
Fixed it.